### ISSUE TITLE
[3.x]: craft\commerce\errors\PaymentSourceException: Could not create the payment source

### ISSUE THREAD

**Issue by chandamotiyani** [craftcms/commerce#3190]
### Description
I recently upgraded Craft Commerce to 3.4.22.1 and getting this error **craft\commerce\errors\PaymentSourceException: Could not create the payment source** on our live environment 

### Craft CMS version

3.8.13

### Craft Commerce version

3.4.22.1

### PHP version

7.2.34

### Installed plugins and versions

```
{
    "require": {
        "ext-json": "*",
        "craftcms/cms": "3.8.13",
        "vlucas/phpdotenv": "^2.4.0",
        "verbb/navigation": "1.4.32",
        "craftcms/commerce": "3.4.22.1",
        "spicyweb/craft-neo": "2.13.18",
        "craftcms/redactor": "2.10.12",
        "studioespresso/craft-navigate": "2.3.0",
        "craftcms/aws-s3": "1.3.2",
        "verbb/super-table": "2.7.5.1",
        "nystudio107/craft-typogrify": "^1.1",
        "verbb/wishlist": "1.4.18",
        "league/flysystem-aws-s3-v3": "^1.0.13",
        "phpoffice/phpspreadsheet": "^1.8",
        "jalendport/craft-queuemanager": "^1.2",
        "nystudio107/craft-templatecomments": "1.2.0",
        "verbb/cp-nav": "3.1.0",
        "solspace/craft-freeform": "3.13.29",
        "yiisoft/yii2-redis": "~2.0.0",
        "nystudio107/craft-seomatic": "3.4.56",
        "symfony/process": "5.2.1",
        "ostark/craft-async-queue": "2.3.0",
        "studioespresso/craft-google-shopping-feed": "1.3.1",
        "isev-ltd/contact-form-validation": "0.0.1",
        "craftcms/element-api": "2.8.6.1",
        "mmikkel/cp-field-inspect": "1.4.4",
        "doublesecretagency/craft-inventory": "2.1.1",
        "studioespresso/craft-seeder": "3.3.1",
        "ngraf/slack-codeception-extension": "^1.8",
        "newism/craft-rollbar": "^1.3",
        "dompdf/dompdf": "^2.0"
    },
    "require-dev": {
        "nystudio107/craft-twigprofiler": "1.0.1",
        "am-impact/amcommand": "3.1.4",
        "charliedev/element-map": "1.2.1",
        "codeception/codeception": "^4.0"
    },
    "autoload": {
        "psr-4": {
            "modules\\": "modules/"
        }
    },
    "config": {
        "optimize-autoloader": true,
        "allow-plugins": {
            "craftcms/plugin-installer": true,
            "yiisoft/yii2-composer": true,
            "treeware/plant": true
        }
    },
    "scripts": {
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ]
    },
    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "5.2.1"
        }
    }
}

```

**Comment by lukeholder** [comment]
Hi, can you please put steps to reproduce and also give the full stack trace.

It might be easier to send your logs and steps to reproduce to support@craftcms.com

**Comment by chandamotiyani** [comment]
Have sent the additional details on the email @lukeholder as suggested, expecting some response, thanks in advance.

**Comment by chandamotiyani** [comment]
hi @lukeholder - it looks like its related to the Custom Shipping Method which we are using and not being handled in, I tried to downgrade it to the fix Version 3.4.20.1 (Craft Commerce) as mentioned in the change log but it is not making any difference

**Comment by lukeholder** [comment]
We have got your support ticket and working on it with you. We will continue assisting over our email support. We will keep re-open this ticket if we find a bug in Commerce.
